app/services/yoksis/v4/ogrenci_bilgi_toplama.rb

```diff
@@ -6,7 +6,7 @@ module V4
       # client = Services::Yoksis::V4::OgrenciBilgiToplama.new
       class OgrenciBilgiToplama
         def initialize
-          wsdl = if Rails.env.production?
+          wsdl = if Rails.env.production? || Rails.env.beta?
```

Comment:
@msdundar, @roktas beta production'a yakın olmasını istiyorsak böyle olmalı sanırım. Öneriniz var mı?

Reply:
mevcut koda göre yan etki olmaması için böyle olmalı.  ama bu tür switchlerin sayısı kodda çok artarsa bir şeyleri yanlış düşünüyoruz demektir.